README.rst

```diff
@@ -201,13 +201,18 @@ This project includes a set of ``ScopeManager`` implementations under the ``open
 
    from opentracing.scope_managers import ThreadLocalScopeManager
 
-There exist implementations for ``thread-local`` (the default instance of the submodule ``opentracing.scope_managers``), ``gevent``, ``Tornado`` and ``asyncio``:
+There exist implementations for ``thread-local`` (the default instance of the submodule ``opentracing.scope_managers``), ``gevent``, ``Tornado``, ``asyncio`` and ``contextvars``:
 
 .. code-block:: python
 
    from opentracing.scope_managers.gevent import GeventScopeManager # requires gevent
    from opentracing.scope_managers.tornado import TornadoScopeManager # requires tornado<6
-   from opentracing.scope_managers.asyncio import AsyncioScopeManager # requires Python 3.4 or newer.
+   from opentracing.scope_managers.asyncio import AsyncioScopeManager # fits for old asyncio applications, requires Python 3.4 or newer.
+   from opentracing.scope_managers.contextvars import ContextVarsScopeManager # for asyncio applications, requires Python 3.7 or newer.
+
+
+**Note** that for asyncio applications it's preferable to use ``ContextVarsScopeManager`` instead of ``AsyncioScopeManager`` because of automatically parent span propagation to children coroutines, tasks or scheduled callbacks.
```

Comment:
```suggestion
**Note** that for asyncio applications it's preferable to use ``ContextVarsScopeManager`` instead of ``AsyncioScopeManager`` because of automatic parent span propagation to children coroutines, tasks or scheduled callbacks.
```

Reply:
thanks!